priv/gettext/fr/LC_MESSAGES/default.po

```diff
@@ -516,13 +516,13 @@ msgstr "Total"
 #: lib/teslamate_web/templates/geo_fence/form.html.leex:118
 msgid "There is <strong>%{n} charging session</strong> at this location for which no costs have been added yet."
 msgid_plural "There are <strong>%{n} charging sessions</strong> at this location for which no costs have been added yet."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Il existe une <strong>% {n} session de charge </strong> à cet endroit pour laquelle aucun coût n'a encore été ajouté."
```

Comment:
```suggestion
msgstr[0] "Il existe <strong>%{n} session de charge </strong> à cet endroit pour laquelle aucun coût n'a encore été ajouté."
```
Wouldn't it be "Il existe une 1 session de charge" otherwise?